lib/action_subscriber/configuration.rb

```diff
@@ -85,7 +85,13 @@ def initialize
         'text/plain' => lambda { |payload| payload.dup }
       }
 
-      self.error_handler = lambda { |error, env_hash| raise }
+      self.error_handler = lambda do |error, env_hash|
+        logger = ::ActionSubscriber::Logging.logger
```

Comment:
will it cause a problem to memoize this value here? What if the user's code changes the logger after this file is loaded?